site/components/pages/Retirements/List/Item.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC } from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
+
+import { Text } from "@klimadao/lib/components";
+
+import { KlimaRetire } from "@klimadao/lib/types/subgraph";
+import { concatAddress, getTypeofTokenByAddress } from "@klimadao/lib/utils";
+import ArrowForwardIcon from "@mui/icons-material/ArrowForwardOutlined";
+import { LeafIcon } from "./Leaf";
+
+import * as styles from "./styles";
+
+type Props = {
+  retirement: KlimaRetire;
+};
+
+export const RetirementItem: FC<Props> = (props) => {
+  const { retirement } = props;
+  const { locale } = useRouter();
+
+  const retirementDate = new Date(parseInt(retirement.timestamp) * 1000); //expects milliseconds
+  const formattedDate = new Intl.DateTimeFormat(locale, {
+    dateStyle: "full",
+  }).format(retirementDate);
+  const typeOfToken = getTypeofTokenByAddress(retirement.offset.tokenAddress);
+  // typeOfToken can be undefined! Due to incorrect tokenAddress received from subgraph - Why?
+
+  const url = `/retirements/${retirement.beneficiaryAddress}/${
+    Number(retirement.index) + 1
+  }`;
+
+  return (
+    <Link href={url} passHref>
```

Comment:
Another design suggestion: would be cool to number them. So they would count from most recent (highest number) to lowest (1)

Reply:
Done